policy-engine/src/graph.rs

```diff
@@ -73,12 +74,38 @@ pub(crate) fn index(
         }))]
     };
 
-    let plugin_params = req.query().to_owned();
+    // TODO(steveeJ): take another look at the actix-web docs for a method that
+    // provides this parameters split.
+    let plugin_params = req
+        .query_string()
+        .to_owned()
+        .split('&')
+        .map(|pair| {
+            let kv_split: Vec<&str> = pair.split('=').collect();
+
+            let value = if kv_split.len() < 2 {
```

Comment:
Not sure if easier to read, but I think this one can be also written as `kv_split.get(1).unwrap_or_else(|| trace(); "")`. Assuming both are feasible, keep the one you prefer.

Reply:
Your suggestions is semantically equivalent in our use-case and avoids the if/else syntax :+1: